src/constants/types/TodoTypes.ts

```diff
@@ -1,6 +1,11 @@
+import { IconTypes } from "components/Icon/Icon";
+
 export interface BasicToDo {
   id: number;
   title: string;
+  priority: "High" | "Medium" | "Low";
+  category: "lifestyle" | "work" | "house" | "car" | "children" | "science";
+  categoryIcon: IconTypes;
 }
 
 export interface TodoType extends BasicToDo {
```

Comment:
You should be consequent with the naming convention you use (Todo/ToDo)

Reply:
You also should create Category Interface and Priority interface